src/SequenceManager.cc

```diff
@@ -327,16 +339,68 @@ void SequenceManager::rename_sequence(UniqueId unique_id, const SequenceName& ne
         throw Error(gul14::cat("Cannot rename folder ", old_path.string(),
             " to ", new_path.string(), ": ", error.message()));
     }
+
+    // commit to local repository
+    gl_.add_files({gul14::cat((const std::string&) new_path, "/*")});
+    gl_.commit(gul14::cat("Rename ", old_seq_on_disk.name.string(), " to ", new_name.string()));
 }
 
 void SequenceManager::rename_sequence(Sequence& sequence, const SequenceName& new_name)
-    const
 {
     rename_sequence(sequence.get_unique_id(), new_name);
     sequence.set_name(new_name);
 }
 
-void SequenceManager::store_sequence(const Sequence& seq) const
+void SequenceManager::store_sequence(const Sequence& seq)
+{
+    _store_sequence(seq);
+
+    // detect what has changed in the sequence
+    const auto dir_name = make_sequence_filename(seq.get_name(), seq.get_unique_id());
+    std::string git_msg{"Store changes:"};
+    auto stats = gl_.status();
+    for(const auto& elm: stats)
+    {
+        // filter for changes in sequence
+        if (gul14::starts_with(elm.path_name, (std::string) (path_ / dir_name)))
+        {
+            if (elm.changes == "new file")
+            {
+                gl_.add_files({elm.path_name});
+                git_msg += gul14::cat("\n", "- create ",elm.path_name);
+            }
+            else if (elm.changes == "modified")
+            {
+                gl_.add_files({elm.path_name});
+                git_msg += gul14::cat("\n", "- modify ",elm.path_name);
+            }
+            //TODO: file number is changing. Figure out which file is deleted
+            //INFO: Maybe git figures it out on its own and tag it with "renamed"
+            else if (elm.changes == "deleted")
+            {
+                gl_.add_files({elm.path_name});
```

Comment:
Really? We can use `add_files()` to delete files from the repo?

Reply:
No, we cant (sadly). I fixed this in later commits